src/sandstorm/run-bundle.c++

```diff
@@ -1511,8 +1512,14 @@ private:
           backendStartTime = getTime();
         }
         if (mongoDied) {
+          // If Mongo died, let's try doing a repair. This way, every second mongo starting is a
```

Comment:
It would be nice to detect more precisely whether Mongo is in need of repair. There are a lot of other reasons why mongo sometimes dies, e.g. out of disk space, and doing a repair in those cases seems risky.